Makefile

```diff
@@ -48,11 +48,16 @@ setup: ## install dependencies
 	@go get -u github.com/gordonklaus/ineffassign
 	@go get -u github.com/client9/misspell/cmd/misspell
 	@go get -u github.com/lk4d4/vndr
+	@go get -u github.com/stevvooe/protobuild
 
-generate: bin/protoc-gen-gogoswarm ## generate protobuf
+generate: protos
 	@echo "🐳 $@"
 	@PATH=${ROOTDIR}/bin:${PATH} go generate -x ${PACKAGES}
 
+protos: bin/protoc-gen-gogoswarm ## generate protobuf
```

Comment:
Unless `protos` is an actual file that gets generated, it should be added to the `.PHONY` target.